tools/streamer.py

```diff
@@ -0,0 +1,8 @@
+import os
+from time import sleep 
+
+while True:
+  os.system("/home/pi/companion/scripts/start_video.sh $(cat /home/pi/vidformat.param)") 
```

Comment:
You can check if both files exist (`os.path.isfile`) before calling `os.system` and printing a message while returning exist with a number different of zero._better safe than sorry_